verticapy/utilities.py

```diff
@@ -284,8 +282,6 @@ def drop(
         result = True
     else:
         result = True
-    if conn:
```

Comment:
Can we also remove `conn` from line 136?

Reply:
Yes.